src/applications/vaos/new-appointment/components/AppointmentsRadioWidget.jsx

```diff
@@ -6,14 +6,14 @@ import PropTypes from 'prop-types';
  * code to disable certain options. This isn't currently supported by the
  * form system.
  */
-export default function SimpleRadioWidget({
+export default function AppointmentsRadioWidget({
```

Comment:
Turns out combining the functionalities of the simple radio widget and descriptive radio widget was simpler than expected so I combined them into one component instead and renamed it to AppointmentsRadioWidget.